### DISCUSSION TITLE
Add Mode Developer Mainnet to @wagmi/chains

### DISCUSSION THREAD

**Discussion by nanthanwa** [wevm/wagmi#3262] [Ideas]
https://docs.mode.network/mode-developer-mainnet/mode-dev-mainnet-information

**Comment by tmm** [comment]
Feel free to create a pull request on [Viem](https://github.com/wevm/viem). [More info.](https://github.com/wevm/viem/blob/main/.github/CONTRIBUTING.md#adding-a-chain)
